### ISSUE TITLE
[GrindstoneGui] setCancelled not working properly

### ISSUE THREAD

**Issue by Nearata** [stefvanschie/IF#264]
**Minecraft Version**: paper-1.18.2-388
**Plugin Version**: 0.10.7

## Code

```java
grindstonegui.setOnGlobalClick(e -> e.setCancelled(true));
grindstonegui.setOnGlobalDrag(e -> e.setCancelled(true));
```

## Current behaviour

https://imgur.com/a/RjXsSNP

## Expected behaviour

The `clicks` and `drags` should be cancelled, like it is with `InventoryClickEvent`.

## Note

The `ChestGui` is not affected by this bug, if is as such.

**Comment by stefvanschie** [comment]
Thank you for the report, I'll look into it.

**Comment by stefvanschie** [comment]
This will be fixed in version 0.10.8.
